R/get_events.R

```diff
@@ -9,7 +9,7 @@
 #'  * proposed
 #'  * suggested
 #'  * upcoming
-#'
+#' @param fields Character or characters separated by comma (e.g "event_hosts" or "event_hosts, past_event_count_inclusive").
```

Comment:
I tried passing `"event_hosts, past_event_count_inclusive"` and it did not work...  I don't see "past_event_count_inclusive" on the list of additional fields: https://www.meetup.com/meetup_api/docs/2/events/

Reply:
Thank you for reviewing this. The additional fields are stored in `resource` - last attribute in the tibble, so the following works for me:

`urlname <- "rladies-ames"`
`#get past meetings of a single group`
`past_meetings <- get_events(urlname = urlname, event_status = "past", fields = "event_hosts, past_event_count_inclusive", api_key = api_key)`
`# get events hosts of single past meeting `
`single_event <- past_meetings$resource[[1]]$event_hosts`

These additional fields are not available in v2 of the meetup API. You can see them in v3 here: https://www.meetup.com/meetup_api/docs/:urlname/events/#list

Reply:
The `past_event_count_inclusive` field is not returned because it's not a valid field:

```
> names(past_meetings$resource[[1]])
 [1] "created"                "duration"               "id"
 [4] "name"                   "date_in_series_pattern" "status"
 [7] "time"                   "local_date"             "local_time"
[10] "rsvp_close_offset"      "updated"                "utc_offset"
[13] "waitlist_count"         "yes_rsvp_count"         "venue"
[16] "group"                  "link"                   "description"
[19] "how_to_find_us"         "visibility"             "pro_is_email_shared"
[22] "event_hosts"
> past_meetings$resource[[1]]$past_event_count_inclusive
NULL
```

Reply:
Are you able to see it in your results?  I see that it's listed in v3 but not v2 so I wonder if that's causing some issue.

Reply:
No, I don't see `past_event_count_inclusive`. This looks like a problem from within the API itself, because, I tried `group_past_event_count` and it returned a value under `group` list in `resource`.

`past_meetings <- get_events(urlname = urlname, event_status = "past", fields = "event_hosts, group_past_event_count", api_key = api_key)`

`names(past_meetings$resource[[1]]$group)`

 So I guess, it would be better to take out `past_event_count_inclusive` for now.

Reply:
Where the optional `fields` parameters are in the range of 30 - 40 different parameters for `get_events()`, would you like to hardcode say, all 40 parameters in the `fields` description? I see that the parameters are many in v3.